addons/rearm/XEH_postInit.sqf

```diff
@@ -1,11 +1,22 @@
 #include "script_component.hpp"
 
+// need these events before enabled check for zeus rearm
+[QGVAR(rearmEntireVehicleSuccessEH), LINKFUNC(rearmEntireVehicleSuccess)] call CBA_fnc_addEventHandler;
+[QGVAR(rearmEntireVehicleSuccessLocalEH), LINKFUNC(rearmEntireVehicleSuccessLocal)] call CBA_fnc_addEventHandler;
+
+if !GVAR(enabled) exitWith {};
+
 GVAR(hardpointGroupsCache) = [] call CBA_fnc_createNamespace;
 GVAR(configTypesAdded) = [];
 ["ace_settingsInitialized", {
     TRACE_2("settingsInit",GVAR(level),GVAR(supply));
-    ["LandVehicle", "Init", {_this call FUNC(initSupplyVehicle)}, true, ["StaticWeapon"], true] call CBA_fnc_addClassEventHandler;
+    ["AllVehicles", "Init", {_this call FUNC(initSupplyVehicle)}, true, ["Man", "StaticWeapon"], true] call CBA_fnc_addClassEventHandler;
```

Comment:
Humans can be supply vehicles? :thinking:

Reply:
Humans would be `CAManBase`, this enables rabbits and snakes to be supply vehicles, too.

Reply:
_Having rabbits and snakes carry ammo for me wouldn't be all that bad._

Reply:
But `Man` is excluded. This takes care of both humans and animals, doesn't it?

Reply:
Oh! That is exclusion, my bad!